testsuite/features/allcli_system_group.feature

```diff
@@ -70,6 +84,22 @@ Feature: Manage a group of systems
     And I should see a "General System Configuration" text
     And the "locale" formula should be unchecked
 
+@centos_minion
+  Scenario: Apply the highstate to the group
+    Given I am authorized as "admin" with password "admin"
+    When I follow "Home" in the left menu
```

Comment:
`When I follow the left menu "Home > Systems > System Groups"`

Reply:
I used "and i'm the groups page"  which is used earlier in the test. i missed that it was used.